Hyper/MathUtiils/GeomPorting.cs

```diff
@@ -4,11 +4,23 @@ namespace Hyper.MathUtiils;
 
 internal static class GeomPorting
 {
+    /// <summary>
+    /// Gets the position of a point in euclidean space from a point in non-euclidean space.
```

Comment:
Maybe let's write it as `ports a point in euclidean space to non-euclidean space`